catalog/imaging/catalog-imaging-transformer/src/main/java/ddf/catalog/transformer/nitf/NitfInputTransformer.java

```diff
@@ -0,0 +1,227 @@
+                    (text, index) -> handleSegmentHeader(metacard, text, TextAttribute.values()));
+
+            //handles symbol segments (nitf 2.0 only)
+            handleSegment(parsingStrategy.getSymbolSegmentHeaders(),
+                    (symbol, index) -> handleSegmentHeader(metacard, symbol,
+                            SymbolAttribute.values()));
+
+            //handles label segments (nitf 2.0 only)
+            handleSegment(parsingStrategy.getLabelSegmentHeaders(),
+                    (label, index) -> handleSegmentHeader(metacard, label,
+                            LabelAttribute.values()));
+
+            // Set GEOGRAPHY from discovered polygons
+            if (polygonList.size() == 1) {
+                metacard.setAttribute(Metacard.GEOGRAPHY, polygonList.get(0).toText());
+            } else if (polygonList.size() > 1) {
+                Polygon[] polyAry = polygonList.toArray(new Polygon[0]);
+                MultiPolygon multiPolygon = GEOMETRY_FACTORY.createMultiPolygon(polyAry);
+                metacard.setAttribute(Metacard.GEOGRAPHY, multiPolygon.toText());
+            }
+        } catch (ParseException e) {
+            throw new CatalogTransformerException(e);
+        }
+    }
+
+    private <T extends CommonNitfSegment> void handleSegment(List<T> segmentList,
+            BiConsumer<T, Integer> segmentConsumer) {
+        int i = 0;
+
+        for (T segment : segmentList) {
+            segmentConsumer.accept(segment, i);
+            i++;
+        }
+    }
+
+    private void handleNitfHeader(Metacard metacard, NitfFileHeader header) {
+        Date date = (Date) NitfHeaderAttribute.FILE_DATE_AND_TIME.getAccessorFunction()
+                .apply(header);
+
+        metacard.setAttribute(new AttributeImpl(Metacard.TITLE, header.getFileTitle()));
+        metacard.setAttribute(new AttributeImpl(Metacard.MODIFIED, date));
+        metacard.setAttribute(new AttributeImpl(Metacard.CREATED, date));
+        metacard.setAttribute(new AttributeImpl(Metacard.EFFECTIVE, date));
+        handleSegmentHeader(metacard, header, NitfHeaderAttribute.values());
+    }
+
+    private void handleImageSegmentHeader(Metacard metacard,
+            NitfImageSegmentHeader imagesegmentHeader, List<Polygon> polygons) {
+
+        handleSegmentHeader(metacard, imagesegmentHeader, ImageAttribute.values());
+
+        // handle geometry
+        if ((imagesegmentHeader.getImageCoordinatesRepresentation()
+                == ImageCoordinatesRepresentation.GEOGRAPHIC) || (
+                imagesegmentHeader.getImageCoordinatesRepresentation()
+                        == ImageCoordinatesRepresentation.DECIMALDEGREES)) {
+            polygons.add(getPolygonForSegment(imagesegmentHeader, GEOMETRY_FACTORY));
+        } else if (imagesegmentHeader.getImageCoordinatesRepresentation()
+                != ImageCoordinatesRepresentation.NONE) {
+            LOGGER.info("Unsupported representation: {}",
```

Comment:
Might be more valuable at WARN, INFO tends to get lost in the logs. Also could we update the message to state the consequences of an Unsupported Representation.